routes.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"github.com/gorilla/mux"
+	"net/http"
+)
+
+type Route struct {
+	Name        string
+	Method      string
+	Pattern     string
+	HandlerFunc http.HandlerFunc
+}
+
+func newRouter(sf *Sunfish) *mux.Router {
+	var routes = []Route{
+		Route{
+			"Index",
+			"POST",
+			"/api/siafile/",
+			sf.AddFile,
+		},
```

Comment:
IIRC the `Route` type is optional here. Fine for now though. Also, use keyed fields.